src/lib/instrument-listen.js

```diff
@@ -77,6 +79,21 @@ module.exports = class InstrumentListen {
 				res.statusCode = isValidErrorStatus ? statusCode : 500;
```

Comment:
**question**: was it intentional to only fallback to the default error handler after setting the status code? the code sample in your linked express documentation shows the call to `next` happening before the code is set. I have no idea if setting the status code this late has any bearing: I guess not if the headers have already been sent?!

Reply:
Yeah I'm pretty sure it won't have an impact at all because HTTP sends the headers after the status code, so if headers are sent then status has been too. I wonder if I should move it just to be safe? 🤔

Reply:
I'm _sure_ it's fine 😄